### PR TITLE
Change footer information to use 'Lynch Media'

### PR DESCRIPTION
**Pull request by jwade1327**
Screen Shot 2022-03-14 at 10 29 13 AM

### CODE REVIEW

**Comment by jwade1327** [comment]
[Create Newsletter for new DDT micro-site](https://trello.com/c/cQCF65Xv/211-create-newsletter-for-new-ddt-micro-site)

File: tenants/multi/templates/today-medical-design-development.marko [hunk 1]
```diff
@@ -122,7 +122,13 @@ $ const socialMediaProviders = config.getAsArray('mdd.socialMediaLinks');
     />
 
     <!-- Footer/OptOut -->
-    <common-default-footer-block />
+    <common-default-footer-block
+      unsubscribe-cloud-page-id=854
+      company-name="Lynch Media"
+      please-contact-email="david@ien.com"
```
Comment (Shinsina) [inline tenants/multi/templates/today-medical-design-development.marko:128]: Is this the correct contact-email? The other instance of this footer override I can find is using tom@cannabisequipmentnews.com

Comment (jwade1327) [reply tenants/multi/templates/today-medical-design-development.marko:128]: @Shinsina david@ien.com is the correct email (I missed that change when I copied over the footer from cen newsletter, which is what they said it should be a copy of)

Comment (Shinsina) [reply tenants/multi/templates/today-medical-design-development.marko:128]: @jwade1327 The text in the screenshot still says Tom Lynch over (what I presume) to be David Mantey.

**Pull request merged**